index.html

```diff
@@ -109,6 +109,7 @@ <h1 class="brand-heading"><span class="light orange">Vol</span><span class="eure
                     <h3>October 29th -  31st</h3>
 
                     <h3><a class="page-scroll" href="#venue">Min H. Kao Building<br>UTK</a></h3>
```

Comment:
Should hybrid event details be above the venue? Also should we clarify the venue being for in person somehow?

Reply:
Should i put "(for in person)" besides the title "venue"?